packages/frontend/src/components/voteBox.tsx

```diff
@@ -147,27 +159,67 @@ const VoteBox = ({ isUpvote, closeVote, dataId, isPost }: Props) => {
                             </div>
                         </div>
                     </div>
-                    <div className="epks">
-                        <div
-                            className={epkNonce === -1 ? 'epk chosen' : 'epk'}
-                            onClick={() => setEpkNonce(-1)}
-                        >
-                            Subsidy
+                </div>
+                <div className="white-box">
+                    <div className="info">
+                        <div className="choose-from">
+                            <div className="choices">
+                                <strong
```

Comment:
The `strong` tag has specific meaning in html, for important text. Don't use it for styling text, use css instead.